resources/views/settings/slack.blade.php

```diff
@@ -12,252 +12,22 @@
 
 
 {{-- Page content --}}
-@section('content')
 
-    <style>
-        .checkbox label {
-            padding-right: 40px;
-        }
-    </style>
 
 
-    {{ Form::open(['method' => 'POST', 'files' => false, 'autocomplete' => 'off', 'class' => 'form-horizontal', 'role' => 'form' ]) }}
-    <!-- CSRF Token -->
-    {{csrf_field()}}
 
-    <div class="row">
-        <div class="col-sm-10 col-sm-offset-1 col-md-8 col-md-offset-2">
 
 
-            <div class="panel box box-default">
-                <div class="box-header with-border">
-                    <h2 class="box-title">
-                        <i class="fab fa-slack"></i> {{ trans('admin/settings/general.slack') }}
-                    </h2>
-                </div>
-                <div class="box-body">
+ @livewire('slack-settings-form',  ['setting' => $setting])
```

Comment:
I don't know if it's possible, but Livewire has the ability to route particular URL's *directly* to a Livewire Controller, skipping the regular 'blade' process entirely. Last time I tried that it didn't work quite right, but I'd like to see a stab at this on this PR. Basically it's something like `Route::get(blah, blah, App/Http/Livewire/Something::class)` in the routes - and then the request goes directly to a Livewire controller, skipping everything else. Keeps things simpler - if it works. And, again, for me, last time, it didn't. But it keeps getting a little better each time, so maybe now it is?